doc/source/user_guide_embedding/index.rst

```diff
@@ -50,6 +51,11 @@ By default, an instance of the :class:`Application <ansys.mechanical.core.embedd
 is configured in the same way as Mechanical. To customize an instance, see
 :ref:`ref_embedding_user_guide_configuration`.
 
+Diagnosing problems with embedding
+----------------------------------
+In some cases, debugging why the embedded Mechanical instance is not working requires additional logging.
+See the :ref:`ref_embedding_user_guide_logging` for instructions on how to configure logging.
+
 Running PyMechanical embedding scripts inside Mechanical with IronPython
 ------------------------------------------------------------------------
 If your PyMechanical embedding script does not use any other third-party Python package, such as `NumPy`,
```

Comment:
I can't leave comments in the lower lines. However, you never want to start a sentence before one code block and then pick it up after subsequent code blocks. It really reduces readability/comprehension. First, in line 62-63, I think the "With scripting ..." is part of the sentence in line 62 (?). Otherwise, it's a sentence fragment. Second, I'd recommend this sentence to introduce the first code block: "This script can be converted to a Python file, such as ``file.py``: Then, before the second code block, use something like this: The Python file would contain this content: Lastly, before the sections for Windows and Linux, introduce them with: You can run this file inside Mechanical using the command line.

Reply:
@PipKat this is really good feedback. Looking back at this I think it is much better to break sentences at code fragments